apps/omg_watcher/lib/db/transaction.ex

```diff
@@ -69,15 +69,17 @@ defmodule OMG.Watcher.DB.Transaction do
     query =
       from(
         tx in __MODULE__,
-        distinct: tx.txhash,
         left_join: output in assoc(tx, :outputs),
         left_join: input in assoc(tx, :inputs),
         where: output.owner == ^address or input.owner == ^address,
-        order_by: [desc: tx.blknum, desc: tx.txindex],
-        limit: ^limit
+        order_by: [desc: tx.txindex]
```

Comment:
is the removal of `desc: tx.blknum` here intentional?

Reply:
no, that's a bug